src/api/curriculumApi.ts

```diff
@@ -97,6 +98,14 @@ interface CrossSubjectTopic {
   'lenke-til-beskrivelse': string;
 }
 
+async function curriculumFetch(
+  path: string,
+  context: Context,
+  options?: RequestOptions,
+): Promise<Response> {
+  return fetch(path, context, { timeout: 10000, ...options });
```

Comment:
Synes ti sekund høres veldig lenge ut...

Reply:
Hvor lenge tenker du?
På et tidspunkt så kommer `@defer` direktivet som kan brukes til akkurat det vi ønsker å gjøre med disse feltene, men det er ikke klart ennå https://graphql.org/blog/2020-12-08-improving-latency-with-defer-and-stream-directives og da stemmer jeg for ganske lang timeout i alle fall (Om noen i det hele tatt).

Reply:
`@defer` ser jo veldig nyttig ut, men inntil videre er timeout den beste løsninga.
Eg berre tenker at om en artikkel har feks ti grepkoder vil det føre til ti kall og dermed en timeout på 100 sekund. Dei aller fleste eksterne kall er godt under sekundet så en timeout på ett sekund burde holde for det meste. Men kanskje vi skal returnere tomt i stedet for å kaste exception? Det vil gjerne føre til færre feilsituasjoner.

Reply:
Antar at det skjer async (og om det ikke gjør det så burde vi endre det før noe annet), så timeouten vil nok i praksis være 10 sekunder uansett, men jeg kan godt minske den.
Jeg syns 1 sekund var litt i minste laget igjen siden det kan sannsynligvis skje tidvis under helt vanlig load. Hva med 3 sekunder? :smile:

Reply:
Godt poeng. Tenkte ikkje på at det var async. Men tre sekunder høres ut som en bra mellomting.